### ISSUE TITLE
Decision on Venue

### ISSUE THREAD

**Issue by nithlack-zz** [honk-ci/honkcon#1]
**Venue Musts**

1. Wifi
    - Intranet
2. Rural 
3. Meeting Rooms
    - Digital Presentations
4. Hold 150 people

**Tasks**
- [ ] Contact Venue
- [ ] Send Overview / Rundown on the cons findings and decision 
- [ ]  Review Responses / Reaction 
- [ ] Book Venue

**Comment by nithlack** [comment]
**Venue Musts**

1. Wifi
    - Intranet
2. Rural 
3. Meeting Rooms
    - Digital Presentations

**Comment by nithlack-zz** [comment]
**Venue 1- Front Runner**

https://www.audubon-center.org/

https://www.audubon-center.org/pdf_links/FY19_Lodging-Conference-Pricing.pdf

**Cost**
---

**Room** 
```
Entire Crosby Lodge Rental (includes two meeting rooms, one large lounge and 14 dorm rooms for a lodging capacity of up to 112 people. No kitchen facilities so meal service is required): $1,855.00 per night lodging only; meals calculated separately (see meal pricing on another side)
```

**Food**
40 a person a day

**Comment by nithlack-zz** [comment]
@jeefy update?

**Comment by jeefy** [comment]
You've been CC'd on the email.

tl;dr in talks with venue, need clarification on a few points before we do
a contract. Dates are available for us.
